docs/application/web/guides/data/data-filter.md

```diff
@@ -351,44 +351,44 @@ The following table lists the filter attributes related to the `findMessage()` m
 | `hasAttachment`  | Yes                        | No                               |
 | `isHighPriority` | Yes                        | No                               |
 | `subject`        | Yes                        | No                               |
-| `isResponseTo`   | No                         | No                               |
-| `messageStatus`  | No                         | No                               |
+| `inResponseTo`   | Yes                        | No                               |
+| `messageStatus`  | Yes                        | No                               |
 | `attachments`    | No                         | No                               |
 
-The following table lists the filter attributes related to the `findConversations()` method.
+The following table lists the attributes by which `MessageConversation` objects may be filtered.
 
-**Table: Filter attributes for finding conversations**
+<span id="conversation-filters-table">**Table: Filter attributes for finding conversations**</span>
```

Comment:
When you update this PR, please remove this span tag.  
Only **\*\*Table: Filter attributes for finding conversations\*\*** is enough.

Reply:
If you want to create a link to a point which is not a title, you can use `<a name="conversation-filters-table"></a>`

For example, if you check web/guides/multimedia/media-controller.md file, 
you'll see "send_custom_commands" is inserted as below to jump to the one of list item.

![image](https://user-images.githubusercontent.com/9958374/52384594-80d4df80-2ac1-11e9-9848-a31d6e8951b4.png)

Would you please use `<a name>` to link to the tables?

Reply:
I have applied the suggested change - anchors are now added with <a name="..."></a>.